src/contracts/Xmetric.sol

```diff
@@ -5,9 +5,9 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Pausable.sol";
 
-contract Xmetric is ERC20("xMetric", "xMETRIC"), ERC20Pausable, Ownable {
+contract Xmetric is ERC20("Xmetric", "xMETRIC"), ERC20Pausable, Ownable {
```

Comment:
it needs to be xMetric!  is this our linter?

Reply:
I updated this because the tests were failing, it was expecting Xmetric. We deployed Xmetric to polygon as well

Reply:
so this is deployed to polygon let's keep it 
```contract Xmetric is ERC20("Xmetric", "xMETRIC"), ERC20Pausable, Ownable {```
these two tests need to be updated I think they got mixed
```

    function test_GetTokenName() public {
        console.log("Test ERC20 initialization");
        assertEq(metricToken.name(), "xMetric");
    }

    function test_GetTokenSymbol() public {
        assertEq(metricToken.symbol(), "xMETRIC");
    }
```

Reply:
@thermohaline  Please pull from latest - this should be fixed so you don't need to make any changes